packages/web-react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { AccordionOpenStateType, AccordionProps, AccordionContextProps } from '../../types';
+import { AccordionProvider } from './AccordionContext';
+import { useAccordionStyleProps } from './useAccordionStyleProps';
+
+const Accordion: React.FC<AccordionProps> = (props) => {
+  const { children, elementType: ElementTag = 'section', id, open = [], toggle, ...rest } = props;
+  const { classProps } = useAccordionStyleProps();
+
+  const contextValue: AccordionContextProps = {
+    open,
+    toggle,
+    isItemOpen: (state: AccordionOpenStateType, iid: string) => iid === state || state?.includes(iid),
+  };
+
+  return (
+    <ElementTag className={classProps.root} id={id} {...rest}>
```

Comment:
Transfer props placing should be at the beginning because we do not want the developer to be able to override our props.

```suggestion
    <ElementTag {...restProp} className={classProps.root} id={id}>
```

Reply:
done